app/js/components/manageApps/ActionAddonModal.jsx

```diff
@@ -51,19 +51,20 @@ export default class DeleteModal extends Component {
               className="btn btn-danger"
               data-dismiss="modal"
               id="cancel-btn"
-              onClick={() => handleDelete(app.name, appUuid)}
-            >Delete</button>
+              onClick={() => (action === 'delete')? handleAction(appUuid, app.name): handleAction(appUuid, action)}
+            >{(action === 'delete')? "Delete" : "Stop"}</button>
           </ModalFooter>
         </Modal>
     );
   }
 }
 
-DeleteModal.propTypes = {
+ActionModal.propTypes = {
   app: React.PropTypes.object.isRequired,
```

Comment:
`React.PropTypes` is deprecated. You might want to install `prop-types` as a separate dependency.